src/xenia/gpu/gl4/premake5.lua

```diff
@@ -21,6 +21,7 @@ project("xenia-gpu-gl4")
   includedirs({
     project_root.."/third_party/gflags/src",
   })
+  filter("platforms:Windows")
```

Comment:
Looks like you're disabling this project - could just do this from where it's included instead.